.changeset/sixty-wolves-type.md

```diff
@@ -0,0 +1,5 @@
+---
+'@cloudflare/next-on-pages': patch
+---
+
+Fix broken `__index.prefetch.rsc` rewrites.
```

Comment:
if you could add a bit more context here (what this actually fixes) that'd be nice, but if you feel like this is enough I am ok with it :slightly_smiling_face: